source/grid/tria.cc

```diff
@@ -6867,30 +6867,30 @@ namespace internal
                     // The table defines the vertices of the lines above
                     // see relevant_lines for mapping between line indices and
                     // vertex numbering
-                    const auto &table =
-                      hex->reference_cell()
+                    const auto table =
```

Comment:
```suggestion
                    const auto &table =
```
Keep it a reference.

Reply:
I meant to add a comment to this. The code here is
```
const auto &table = cell->reference_cell()
                        .new_isotropic_child_face_line_vertices(
                          chosen_line_tetrahedron);
```
But the function in question here is declared as
```
  constexpr dealii::ndarray<unsigned int, 12, 4, 2>
  new_isotropic_child_face_line_vertices(
    const unsigned int refinement_choice) const;
```
In other words, it returns an *object*, not a *reference*. When we declare `table` as a reference, what the compiler does is take the returned object, store it as an invisible variable on the stack, and bind the reference to it. The invisible object goes out of scope when the references does ("lifetime extension of the temporary object"). But all of that is unnecessary: Since the function returns an object, we may as well put it into the variable `table` right away, rather than bind a reference to the temporary object.